server/src/main/java/org/jboss/as/server/ApplicationServerService.java

```diff
@@ -104,7 +104,7 @@ public synchronized void start(final StartContext context) throws StartException
         final ServerEnvironment serverEnvironment = configuration.getServerEnvironment();
         final ProductConfig config = serverEnvironment.getProductConfig();
         final String prettyVersion = config.getPrettyVersionString();
-        ServerLogger.AS_ROOT_LOGGER.serverStarting(prettyVersion);
+        ServerLogger.AS_ROOT_LOGGER.serverStarting(prettyVersion, config.getCoolName());
```

Comment:
what about `config.getCoolName()` returning null?

Reply:
not nullable anymore